src/spider/core/Task.hpp

```diff
@@ -215,6 +216,8 @@ class Task {
 
     [[nodiscard]] auto get_outputs() const -> std::vector<TaskOutput> const& { return m_outputs; }
 
+    [[nodiscard]] auto get_arg_buffers() const -> std::optional<std::vector<msgpack::sbuffer>>;
```

Comment:
Add docstring.

Reply:
@coderabbitai Please add an issue to add class docstrings to `Task` and `TaskInput`.